dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/DBTaskAckProcessor.java

```diff
@@ -46,8 +46,10 @@ public void process(Channel channel, Command command) {
                 command.getBody(), DBTaskAckCommand.class);
 
         if (taskAckCommand == null){
+            logger.error("dBTask ACK request command is null");
```

Comment:
Can you add some identity here? such as processInstanceId or taskInstanceId, or name.